### PR TITLE
feat(Waze): add map editor features 

### PR DESCRIPTION
**Pull request by Lulucmy**
Hi,

**Describe the changes in this pull request:**

- Added Waze Map Editor informations (where the user edits, the road name, location...).

<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image 1

image 2

<br>

</details>

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Use previous format

**Comment by Lulucmy** [comment]
Hi,
It shoul be good now

File: websites/W/Waze/presence.ts [hunk 1]
```diff
@@ -23,10 +23,13 @@ const presence = new Presence({
...
 		"/legal/privacy/": "Privacy Policy",
-		"/legal/copyright/": "Copyright Policy"
+		"/legal/copyright/": "Copyright Policy",
+		"/ccp": "Waze for Cities",
+		"/forum/": "Forum",
+		"/press/": "Press"
 	};
 
-let path: string, start: string, end: string;
+let path: string, start: string, end: string, mapelem: string, maploc: string;
```
Comment (Slowlife01) [inline websites/W/Waze/presence.ts:32]: Use camelCase.

File: websites/W/Waze/dist/metadata.json [hunk 2]
```diff
@@ -4,15 +4,22 @@
...
-		"en": "Get driving directions, a live traffic map & road alerts."
+		"en": "Get driving directions, a live traffic map & road alerts.",
+		"fr": "Obtenez des itinéraires, une carte du trafic en direct et des alertes routières."
 	},
 	"url": [
 		"www.waze.com",
 		"beta.waze.com"
 	],
-	"version": "1.1.2",
+	"version": "1.1.3",
```
Comment (Slowlife01) [inline websites/W/Waze/dist/metadata.json:22]:
```suggestion
	"version": "1.2.0",
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/W/Waze/presence.ts (hunk 1 above)
Comment (Lulucmy) [reply websites/W/Waze/presence.ts]: Done

**Review by Slowlife01 — APPROVED** [review]
Assuming you tested the recent change

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**